### DISCUSSION TITLE
I cannot stop a vela app

### DISCUSSION THREAD

**Discussion by raquelhortab** [kubevela/kubevela#5840] [Q&A]
Hi, I realised something was running on port 80 although it does not appear on netstat. I then remembered I tried kubevela during a workshop and that could be what I see on port 80. However, I cannot get the app to stop:

```
$ sudo vela ls
APP         	COMPONENT	TYPE      	TRAITS                 	PHASE  	HEALTHY	STATUS   	CREATED-TIME                 
my-wordpress	wordpress	webservice	gateway,service-binding	running	healthy	Ready:1/1	2023-02-25 19:35:45 +0100 CET
└─          	mysql    	webservice	storage                	running	healthy	Ready:1/1	2023-02-25 19:35:45 +0100 CET
```

```
$ sudo vela delete my-wordpress
error: failed to get application /my-wordpress: an empty namespace may not be set when a resource name is provided
```

Could someone help? I cannot seem to be able to use port 80 now. I thought about uninstalling vela bus I'm afraid I'll have the kubernetes running in the background without being able to stop it (I know nothing about kubernetes)

**Comment by raquelhortab** [comment]
I fixed it by running `velad uninstall`. Luckily I remembered about having installed vela, otherwise I would have gone mad looking for whatever was running. How's that it does not appear on netstat?

**Reply by wonderflow** [reply]
```
$ sudo vela delete my-wordpress
error: failed to get application /my-wordpress: an empty namespace may not be set when a resource name is provided
```

The way that trying to delete is app like this to free the port is right. While the error should not happen, there must be some other error. What's your version?
